charts/seed-controlplane/charts/packet-cloud-controller-manager/templates/packet-ccm-deployment.yaml

```diff
@@ -0,0 +1,71 @@
+---
+apiVersion: {{ include "deploymentversion" . }}
+kind: Deployment
+metadata:
+  name: packet-cloud-controller-manager
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: packet-cloud-controller-manager
+    garden.sapcloud.io/role: controlplane
+spec:
+  replicas: {{ .Values.replicas }}
+  selector:
+    matchLabels:
+      app: packet-cloud-controller-manager
+  template:
+    metadata:
+      labels:
+        app: packet-cloud-controller-manager
+{{- if .Values.podAnnotations }}
+      annotations:
+{{ toYaml .Values.podAnnotations | indent 8 }}
+{{- end }}
+    spec:
+      dnsPolicy: Default
+      hostNetwork: true
+      tolerations:
+        # this taint is set by all kubelets running `--cloud-provider=external`
+        # so we should tolerate it to schedule the packet ccm
+        - key: "node.cloudprovider.kubernetes.io/uninitialized"
+          value: "true"
+          effect: "NoSchedule"
+        - key: "CriticalAddonsOnly"
+          operator: "Exists"
+        # cloud controller manager should be able to run on masters
+        - key: "node-role.kubernetes.io/master"
+          effect: NoSchedule
+      containers:
+      - image: {{ index .Values.images "packet-controller-manager" }}
+        name: packet-cloud-controller-manager
+        command:
+          - "./packet-cloud-controller-manager"
+          - "--cloud-provider=packet"
+          - "--leader-elect=false"
+          - "--allow-untagged-cloud=true"
+          - "--kubeconfig=/var/lib/cloud-controller-manager/kubeconfig"
+        resources:
```

Comment:
Done

Reply:
Done

Reply:
Remove duplicative declaration of resources